test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -727,6 +727,23 @@ def forward(self, a, b, c, d):
         d = torch.randn(2, 3)
         self.run_test(MyModel(), (a, b, c, d))
 
+    def test_tuple_input(self):
```

Comment:
Do we support for tuple of primitives now?

Reply:
yes, added test case